README.md

```diff
@@ -1,6 +1,22 @@
 # keycloak-theme
 VSHN Keycloak IDP Theme
 
-## Local development
+## Local Development
 
-Run `docker compose up` to start a local keycloak with the vshn theme.
+Run `docker-compose up` to start a local keycloak with the vshn theme.
+
+## Deploy to Your Instance
+
+1. Merge your Pull Request with the theme to master
+1. Tag it with
+    ```bash
+    git tag -am v1.x.y
```

Comment:
those flags are probably not needed?
```suggestion
    git tag v1.x.y
```

Reply:
ah, I wanted to write

```
git tag -am v1.x.y v1.x.y
```

I once was told that I should tag with `--annotate` and `--message` and I do it since then. Do you think it's ok with an unannotated tag?

Reply:
For the release it doesn't matter. But you can keep your -am flags, no hard feelings here :)